docs/13_upload.md

```diff
@@ -51,7 +51,8 @@ two different servers differ slightly.
     Click `NEW DANDISET` in the Web application (top right corner) after logging in. 
     After you provide a name and description, the dataset identifier will be created; 
     we will call this `<dataset_id>`.
-1. NWB format:
+
+1. NWB format
```

Comment:
ideally here we should gain some explanation preamble to decide on which one of the two is relevant and that they are "mutually exclusive" ATM in most of the cases (but not always since BIDS already supports NWB for some limited set of data types). 

Overall it boils to down to those sections being more of:

1. NWB files in a DANDI layout (e.g., animal electrophys data)
2. BIDS datasets (e.g., MRI or microscopy data)

Reply:
may be at the cost of indenting entire text more we are doomed to make a generic 

1. Standardize your dataset

    Here could go common description

    1. NWB files in a DANDI layout (e.g., animal electrophys data)
    1. BIDS datasets (e.g., MRI or microscopy data)

Reply:
Good idea.  I have added this description.